viewer/src/viewer/pipeline.rs

```diff
@@ -61,7 +61,8 @@ impl ViewerPipeline {
       .by(scene.by_main_camera_and_self(BackGroundRendering))
       .by(scene.by_main_camera_and_self(ForwardScene{
         lights: &self.forward_lights, 
-        tonemap: &self.tonemap
+        tonemap: &self.tonemap,
+        surface_shading: &PhysicalShading,
```

Comment:
my initial thought is that we could have a per drawcall shading model in forward rendering. The old todo is to get the current shading model for current drawcall. But it's ok to support this in future